packages/core/src/Badge/Badge.tsx

```diff
@@ -57,11 +57,73 @@ const sizes = {
   `,
 }
 
+const badgeStyle = (props) => {
+  const badgeStyles = {
+    primary: {
+      backgroundColor: props.theme.palette.primary.base,
```

Comment:
It wouldn't hurt to destructure out `palette` so you don't have to repeat `props.theme.palette`